lib/mongoid/slug.rb

```diff
@@ -102,6 +104,17 @@ def slug(*fields, &block)
         end
       end
 
+      def default_slug_builder
+        if Mongoid::Slug.slug
+          Mongoid::Slug.slug
+        else
+          #-- Why is it necessary to customize the slug builder?
+          lambda do |cur_object|
+            cur_object.slug_builder.to_url
+          end
+        end
```

Comment:
This thing is a 1-liner avoids re-evaluating `Mongoid::Slug.slug`, so  `Mongoid::Slug.slug || ->(cur_object) { cur_object.slug_builder.to_url }`